example/main.go

```diff
@@ -32,23 +32,13 @@ func main() {
 	}
 
 	logger := zerolog.New(os.Stdout).With().Timestamp().Logger()
-
-	server, err := baseapp.NewServer(
-		config.Server,
-		baseapp.DefaultParams(logger, "exampleapp.")...,
-	)
-	if err != nil {
-		panic(err)
-	}
+	zerolog.DefaultContextLogger = &logger
 
 	cc, err := githubapp.NewDefaultCachingClientCreator(
 		config.Github,
 		githubapp.WithClientUserAgent("example-app/1.0.0"),
 		githubapp.WithClientTimeout(3*time.Second),
 		githubapp.WithClientCaching(false, func() httpcache.Cache { return httpcache.NewMemoryCache() }),
-		githubapp.WithClientMiddleware(
-			githubapp.ClientMetrics(server.Registry()),
```

Comment:
Might be nice to create `metrics.Registry` to pass in here as a demonstration, even if there's nothing exporting the metrics.